pkg/odo/cli/pipelines/bootstrap.go

```diff
@@ -97,8 +97,8 @@ func NewCmdBootstrap(name, fullName string) *cobra.Command {
 	initCmd.Flags().StringVar(&o.GitOpsRepoURL, "gitops-repo-url", "", "GitOps repository e.g. https://github.com/organisation/repository")
 	initCmd.Flags().StringVar(&o.GitOpsWebhookSecret, "gitops-webhook-secret", "", "provide the GitHub webhook secret for GitOps repository (if not provided, it will be auto-generated)")
 
-	initCmd.Flags().StringVar(&o.AppRepoURL, "app-repo-url", "", "Application source e.g. https://github.com/organisation/application")
-	initCmd.Flags().StringVar(&o.AppWebhookSecret, "app-webhook-secret", "", "Provide the GitHub webhook secret for Application repository (if not provided, it will be auto-generated)")
+	initCmd.Flags().StringVar(&o.ServiceRepoURL, "service-repo-url", "", "Application source e.g. https://github.com/organisation/application")
```

Comment:
Shouldn't this say "Service source" and reference a "service" url?

Reply:
made the requested changed